model/metrics/dotnet/dotnet-kestrel.yaml

```diff
@@ -14,6 +14,7 @@ groups:
   - id: metric.kestrel.active_connections
     type: metric
     metric_name: kestrel.active_connections
+    stability: experimental
```

Comment:
The document for these is marked as stable https://github.com/open-telemetry/semantic-conventions/blob/main/docs/dotnet/dotnet-kestrel-metrics.md